### PR TITLE
Feat: Headers validator benchmark

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Adds PHPBench, and benchmark for headers validator

## Test Plan

- [x] Add benchmark CI/CD
- [x] Manual run:

image

## Related PRs and Issues

x

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: .github/workflows/bench.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Run Benchmark
+      run: |
+        docker run --rm -v $PWD:/app composer sh -c \
+        "composer install --profile --ignore-platform-reqs && git config --global --add safe.directory /app && composer bench -- --progress=plain"
```
Comment (christyjacob4) [inline .github/workflows/bench.yml:16]: lets run this as a job in one of the existing workflows 
https://github.com/appwrite/appwrite/blob/main/.github/workflows/tests.yml#L102

Comment (Meldiron) [reply .github/workflows/bench.yml:16]: tests.yml runs for every service. This is perfectly fine to run only once. Separation also allows concurrency, I don't see downside in that. Plus it's copy&paste from utopia library, so it follows consistency to have it separated.

Comment (Meldiron) [reply .github/workflows/bench.yml:16]: I could see same discussion about linter - linter could be in tests.yml, but is separate.

Comment (Meldiron) [reply .github/workflows/bench.yml:16]: Removed bench CI/CD

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**